### PR TITLE
Edit indevedual article

### PR DESCRIPTION
**Pull request by DwinaTech**
added category to edit article and made add and edit categories form to be one form

### CODE REVIEW

**Comment by DwinaTech** [comment]
[Edit Article does not have Category](https://trello.com/c/iJ9pCAZL/52-edit-article-does-not-have-category)

File: server/routes/admin/categories/categories.js [hunk 1]
```diff
@@ -29,8 +32,10 @@ router.post('/add', function (req, res, next) {
 router.get('/edit/:categoryId', (req, res) => {
     const {categoryId} = req.params;
     const callback = (error, category) => {
-        res.render("admin-edit-category", {
-            category: category
+        res.render("admin-edit-and-add-category", {
+            category: category,
```
Comment (kabaros) [inline server/routes/admin/categories/categories.js:36]: don't be lazy with variable names - `desTitle` is a bad name, just say `descriptionTitle`

File: server/routes/admin/articles/articles.js [hunk 2]
```diff
@@ -29,16 +29,32 @@ router.post('/add', (req, res) => {
...
+
+
 router.get('/edit/:articleId', (req, res) => {
     const {articleId} = req.params;
-    const callback = (error, article) => {
-        res.render("admin-edit-article", {
-            article: article
-        })
+    const callback = (error, categories) => {
+        callback2 = (error, article) => {
```
Comment (kabaros) [inline server/routes/admin/articles/articles.js:40]: don't call them callback1 and callback2 - one is `categoriesCallback`, the other is `articleCallback`

File: server/routes/admin/categories/categories.js (hunk 1 above)
Comment (kabaros) [reply server/routes/admin/categories/categories.js:36]: you should not pass these "default" values to the View. Just use input `placeholder` to display this information.

Comment (DwinaTech) [reply server/routes/admin/categories/categories.js:36]: do you mean I do not need labels?

Comment (DwinaTech) [reply server/routes/admin/categories/categories.js:36]: I did that just to be logic title for add or edit

**Pull request merged**